app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -446,7 +446,7 @@ public void onDialogDismissed(int dialogId)
                     // Nothing to do, no change made
                 }
             });
-            dialog.show(getFragmentManager(), "color-picker-dialog");
+            dialog.show(getSupportFragmentManager(), "color-picker-dialog");
```

Comment:
Hmm, Android Studio insists on telling me:

```
Cannot resolve method 'show(androidx.fragment.app.FragmentManager, java.lang.String)'
```

Yet it doesn't crash. Weeeeeird.

Reply:
Nevermind it stopped saying this. Guess it was some broken cache stuff lol.